### PR TITLE
[Feature] Item Variant Creation from Quick Entry

### PR DESCRIPTION
**Pull request by suyashphadtare**
- Creation of item variants from quick entry(considering item template already created).

- Already sent frappe PR - [Enable overriding Quick Entry in custom app](https://github.com/frappe/frappe/pull/3519)

out123

### CODE REVIEW

**Comment by mbauskar** [comment]
@patilsangram,

Please check and fix the codecy issue, Closing the PR, for now, please reopen the PR once the issue s are fixed.

**Comment by mnatalia** [comment]
please check that all your labels have possibility to be translated

File: erpnext/public/js/templates/variant_attribute.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<h4>Item Attributes</h4>
+<table class="table table-bordered table-condensed">
+	<thead>
+		<tr>
+			<td>Page No.</td>
```
Comment (rmehta) [inline erpnext/public/js/templates/variant_attribute.html]: make this translatable `{{ __("Page No") }}`

apply to all labels

**Comment by rmehta** [comment]
@suyashphadtare thanks for the contribution!

Please fix translations

Can you also write a simple test case?

**Review by rmehta — CHANGES_REQUESTED** [review]

**Comment by patilsangram** [comment]
@rmehta , @mnatalia  Thanks for the suggestions, I have fixed translation. I will try test cases for Item variant creation from quick entry.

Thanks,

**Comment by patilsangram** [comment]
1. Moved item UI test under stock 

2. Added UI tests for 

- Item Attribute

-  Item template

- Item variant from quick entry

**Comment by rmehta** [comment]
https://github.com/frappe/erpnext/pull/11535

**Pull request closed without merging**